manifests/c/Cimbali/pympress/1.8.3/Cimbali.pympress.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Cimbali.pympress
+PackageVersion: 1.8.3
+PackageLocale: en-US
+Publisher: Cimbali

```

```suggestion
Publisher: Cimbali, Thomas Jost, Christof Rath, Epithumia
```
Must match Control Panel / Apps&Features